src/xdocs/eclipse.xml

```diff
@@ -72,8 +72,20 @@
       </p>
     </section>
 
-    <section name="Suppress Eclipse warnings on resource files">
+    <section name="Suppress Eclipse warnings">
       <p>
+        1) Use Eclipse compiler preferences from repo:<br/><br/>
+        copy
+        <a href=
+          "https://github.com/checkstyle/checkstyle/blob/master/config/org.eclipse.jdt.core.prefs">
+          config/org.eclipse.jdt.core.prefs</a> file
+        to
+        <code>${YOUR_WORKSPACE}/.metadata/.plugins/org.eclipse.core.runtime/.settings</code>
```

Comment:
@romani I didn't even know this location existed in metadata. Is this the one that is global for the entire workspace?
If so, I don't recommend this location and use the project specific location, especially if you have different projects in your workspace. Original comment said this too.

Example for checkstyle: `${YOUR_WORKSPACE}\checkstyle\.settings`.

Reply:
workspace and location of cloned repo could be different.

did changes to wording, pleas recheck.